internal/processmon/processmon.go

```diff
@@ -256,8 +256,8 @@ func (p *ProcessMon) getLaunchProcessEnvVars(procMountPoint string, contextID st
 	return newEnvVars
 }
 
-//LaunchProcess prepares the environment for the new process and launches the process
-func (p *ProcessMon) LaunchProcess(contextID string, refPid int, refNSPath string, rpchdl rpcwrapper.RPCClient, arg string, statsServerSecret string, procMountPoint string) error {
+// LaunchProcess prepares the environment and launches the process
+func (p *processMon) LaunchProcess(contextID string, refPid int, refNSPath string, rpchdl rpcwrapper.RPCClient, arg string, statsServerSecret string, procMountPoint string) error {
```

Comment:
for big arg list, it's more readable to do 

```
func (p *processMon) LaunchProcess(
    contextID string,
    refPid int, 
    refNSPath string, 
    rpchdl rpcwrapper.RPCClient, 
    arg string, 
    statsServerSecret string, 
    procMountPoint string,
) error {
// stuff
}
```

Reply:
whats big ? 80 chars ?